Formula/vdirsyncer.rb

```diff
@@ -67,6 +67,27 @@ def install
 
     bin.install Dir["#{libexec}/bin/*"]
     bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+
+    prefix.install "contrib"
+  end
+
+  SYNC_INTERVAL = 60
+
+  def caveats; <<-EOS.undent
+      The sync intervall in the launchd .plist file is set to #{SYNC_INTERVAL}s
+      and should be adjusted to your when enabled.
+    EOS
+  end
+
+  plist_options :manual => "vdirsyncer"
+
+  def plist
+    p = File.read(prefix/"contrib/vdirsyncer.plist")
```

Comment:
Instead use `prefix.install "contrib/vdirsyncer.plist"` above and then use `inreplace` on that file.